app/controllers/api_controller.rb

```diff
@@ -5,12 +5,12 @@ class ApiController < ApplicationController
   private
 
   def ensure_jwt
-    render plain: 'auth is wrong', status: :unauthorized unless token_match?
+    render plain: 'Access Denied', status: :unauthorized unless token_match?
```

Comment:
I had a look and the GraphQL spec doesn't seem to include auth so I guess we can do what we like here in terms of response.